spec.md

```diff
@@ -1733,6 +1747,44 @@ message NodeServiceCapability {
 
 If the plugin is unable to complete the NodeGetCapabilities call successfully, it MUST return a non-ok gRPC code in the gRPC status.
 
+
+#### `NodeGetInfo`
```

Comment:
Is `NodeGetInfo` guaranteed to return the same result every time, i.e. can node info change over time? @saad-ali what if topology is included?

Either way I'd recommend making the answer to this question explicit.

Reply:
Yeah I think we meant this value to be invariant through life cycle of the node. I will open a PR to make this explicit. 

There may be an exceptional case though - where node is shutdown, modified and rebooted. For example - on EC2 it is possible to change instance type of a instance.  is topology falls under similar case? I am not sure, if we worded explicitly what happens to node-id when a node is modified and rebooted.